common/src/worldgen.rs

```diff
@@ -130,19 +130,56 @@ impl NodeState {
     }
 }
 
+struct VoxelIterable {
```

Comment:
```suggestion
struct VoxelCoords {
```
Maybe? Just a thought.

Reply:
I think that would be an improvement. I'm trying to view it from the point of view of someone typing `for (x, y, z) in VoxelCoords::new(self.dimension) {}` in a new worldgen function, making it something that's simple enough to remember without looking it up.